src/scene/scene.c

```diff
@@ -403,7 +416,8 @@ void sceneCheckPortals(struct Scene* scene) {
     }
 
     if ((fireOrange || fireBlue) && playerIsGrabbing(&scene->player)){
-        playerSetGrabbing(&scene->player, NULL);
+        playerThrowObject(&scene->player);
```

Comment:
In the original game, the primary portal button throws the object, but the secondary button just drops it (as it was before these changes). Can you differentiate the buttons?

Reply:
Sure.

Reply:
Yeah I was wondering the same thing! I took a quick look at the commit history and it looks like it was put in as a quick fix.